setup.cfg

```diff
@@ -38,8 +38,8 @@ install_requires =
   scipy>=1.0.0
   matplotlib>=2.1.2
   pandas>=0.23.0
+  parfive[ftp]>=1.1rc2
```

Comment:
we should remove this before we merge once the CI has passed.

Reply:
I have set it back to the >= 1.0 as 1.1 has not been released as of yet.